src/modules/ekf2/ekf2_main.cpp

```diff
@@ -578,7 +578,13 @@ void Ekf2::run()
 				// check if distance sensor is within working boundaries
 				if (range_finder.min_distance >= range_finder.current_distance ||
 				    range_finder.max_distance <= range_finder.current_distance) {
-					range_finder_updated = false;
+					// use rng_gnd_clearance if on ground
+					if (_ekf.get_in_air_status()) {
+						range_finder_updated = false;
+
+					} else {
+						range_finder.current_distance = _params->rng_gnd_clearance;
```

Comment:
This is a bit picky, but on the Firmware side we should try to use the BlockParam interface for const access. `_rng_gnd_clearance.get()`
There's a mix of usage in ekf2_main so we can update them all later in a separate pass.